.github/flavors-arm.json

```diff
@@ -1,35 +1,35 @@
 [
     {
         "flavor": "opensuse-leap-arm-rpi",
-        "model": "rpi64",
+        "model": "rpi4",
         "worker": "ubuntu-latest",
         "standard": "true"
     },
     {
         "flavor": "opensuse-tumbleweed-arm-rpi",
-        "model": "rpi64",
+        "model": "rpi4",
         "worker": "ubuntu-latest",
         "standard": "true"
     },
     {
         "flavor": "alpine-arm-rpi",
-        "model": "rpi64",
+        "model": "rpi4",
         "worker": "ubuntu-latest",
         "standard": "true"
     },
     {
         "flavor": "ubuntu-arm-rpi",
-        "model": "rpi64",
+        "model": "rpi4",
```

Comment:
For the ubuntu ones, you will also need to update the Dockerfile e.g.

https://github.com/kairos-io/kairos/blob/master/images/Dockerfile.ubuntu-arm-rpi

I'd just change the existing part exclusive for rpi64 to look something like:

```
FROM base AS baseRpi
RUN apt-get update && apt-get install -y linux-firmware-raspi \
    linux-modules-extra-raspi \
    linux-raspi \
    && apt-get clean && rm -rf /var/lib/apt/lists/*
    
 FROM baseRPi AS rpi3
 
FROM baseRPi AS rpi4
```

Reply:
ahhh good catch!